index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
```

Comment:
- [ ] I suggest you update the <title> element to better reflect the page's content. 📝 This will help users understand the purpose of the page.😊

Reply:
- [ ] very good work. Brief, clear, and to the point 🤝